bibeasy/scripts/student_asterisk_helper.py

```diff
@@ -0,0 +1,69 @@
+import xml.etree.ElementTree as ET
+from bibeasy.formatting import STUDENTS
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
maybe you want to add usage help as you described in your PR?

> Export the latest CCV always, keep the old xml around until you confirm the modifications work
Run it through the script (bibeasy/scripts/student_asterisk_helper.py -i [ccv.xml] -o [ccv-modified.xml]) under a bibeasy virtualenv
Revert the xml on CCV-CVC.ca if any error occurs (and of course let me know so I can fix it)

Reply:
I believe the added help information and internal usage section should make sure we don't get stuck in the future for how this is supposed to be used.